user_guide/intro_navigation.rst

```diff
@@ -0,0 +1,110 @@
+details.)
+  
+  
+.. _user-guide-navigation-panel:
+
+Navigation Tools
+----------------
+
+At the very top of the page, there is navigation panel that contains different navigation tools, as shown below.
+
+|
+
+.. image:: ./img/navigation/panel.png
+
+
+.. csv-table::
+  :header: "#","Item","Description","Related Document"
+  :widths: 10, 30, 40,30
+  
+  "1","**Organization Selector**","Switch between organizations when working in  the multi-organization environment.","
+  :ref:`Working with Multiple Organizations <user-guide-getting-started-change-organization>` section of the 
+  *Company Structure and Organization Selector* guide`"
+  "2","**Shortcuts**",Quickly perform the most common actions from anywhere in the system.","
+  :ref:`Shortcuts <user-guide-getting-started-shortcuts>`"
+  "3","**Search**","Find a specific record's 
+  :ref:`View page <user-guide-ui-components-view-pages>`","
+  :ref:`Search <user-guide-getting-started-search>`"
+  "4","**Pinned pages**","Open pages preserved in the state they were pinned in.",
+  ":ref:`Pinned Page <user-guide-ui-components-pinned>`"
+  "5","**User menu**","Access your user profile, calendar, mailbox, and task list. Log out of the system.","
+  :ref:`User Menu <user-guide-getting-started-user-menu>`"
+  "6","**My Emails**","See if you have new and unread emails. Quickly read and reply to most recent emails. 
+  
+  The envelop icon reflect current state of your mailbox - from 
+  the menu under it, you can get to the *My Emails* page, read and reply any of the most recent 5 emails and mark all 
+  the emails as read",""
+  "7","**Hamburger menu**","Access your
+
+  * recently opened pages
+  * favorite pages
+  * most viewed pages","
+  :ref:`History, Favorites and Most Viewed <user-guide-getting-started-history>`"
+  "7","**Help Icon**","Access the User Guide",""
+  "8","**Sidebar panel**","Hosts widgets that provide fast and convenient access to the constantly needed information, 
+  such as recent emails or the task list. ",":ref:`Sidebar Pannel <user-guide-navigation-sidebar-panel>`"
+  
+
+This way, the users can easily access, store and organize any and all the information required to manage the customer 
+relations in the most efficient way.
+
+Flash notifications and Dialog windows
+--------------------------------------
+
+Flash notifications and dialog windows are used by OroCRM when it notifies you about something important.
+
+If there is a need for notification only (e.g. success or failure of an operation, a reminder, etc.), the flash 
+message will appear. They may contain links to more detailed information. 
+If your decision is mandatory , and OroCRM cannot proceed without it, a dialog window will appear and will remain in 
+place until you click a button. They usually pop up to give you a second chance on something very important or 
+irreversible, such as deleting records.
```

Comment:
Edit: If your decision is required, and OroCRM cannot proceed without it, a dialog window will appear and will remain in place until you click a button. They usually pop up in order to warn you about something very important or irreversible, such as deleting records.